.github/workflows/automated-release-tasks.yml

```diff
@@ -31,38 +31,8 @@ jobs:
     needs: verify-should-run
     if: ${{ needs.verify-should-run.outputs.should-run == 'true' }}
     steps:
-      - name: Invoke v5 workflow
-        uses: actions/github-script@v6
+      - name: Create release branch
+        uses: ./.github/workflows/create-release-branch.yml
+        secrets: inherit
         with:
-          github-token: ${{ secrets.SVC_CLI_BOT_GITHUB_TOKEN }}
-          # TODO: remove inputs after April Release; will default to minor
```

Comment:
I read this comment as referring to the `inputs` property on the script object. If you remove that property, then the default value for `release-type` will just be `minor`, which is what we want.
Are you sure we're able to remove this workflow dispatch and just directly call the workflow as a step?

Reply:
Let me make a fork to double check this one 👍

Reply:
Good callout on this - I did need to make changes over with the fork to make this work! Here's a run that calls create-release-branch successfully: https://github.com/randi274/sfdx-scanner/actions/runs/14800941925

Reply:
It doesn't look like that commit has any changes. Is that expected?

Reply:
It was a lot of small commits over there, but this is the one that should work here: https://github.com/forcedotcom/sfdx-scanner/pull/1787/commits/91dd10387be9e2966906724906ed18e425c8c893

And then the code over there should be correct on dev now